src/BizHawk.Client.EmuHawk/MainForm.FileLoader.cs

```diff
@@ -25,7 +25,7 @@ private enum LoadOrdering
 			LegacyMovieFile
 		}
 
-		public struct FileInformation
```

Comment:
I did not mean to include this useless commit with this PR, sloppy branch management